modules/installation/con-airgapped-requirements-orchestrator.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies
+// assembly-install-rhdh-orchestrator-helm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="con-airgapped-requirements-orchestrator_{context}"]
+= Resource requirements for air-gapped environments
+
+You can deploy the Orchestrator flavor of {product} in air-gapped environments by ensuring that all container images, and Helm charts required by {product} and SonataFlow are available internally.
+
+For workflow orchestration in air-gapped environments, the following resources are recommended:
+
+[cols="2,2", options="header"]
+|===
+| Resource component
+| Recommended resources
+
+| RHDH Orchestrator plugin
+| 2 vCPUs, 2–4 GiB memory
+
+| SonataFlow runtime
+| 1 vCPU, 1–2 GiB memory
+
+| Knative Serving and Eventing
+| 1–2 vCPUs, 1.5 GiB memory
+
+| PostgreSQL or External DB
+| 1 vCPU, 1 GiB memory
+
+| Persistent storage
+| 5 GiB (minimum), adjust based on workflow logs and artifacts
+|===
```

Comment:
Just curious, where are these recommended values coming from? I guess these should not be only for air-gapped environments, but for any env requiring workflow orchestration, no?